CONTRIBUTING.md

```diff
@@ -82,9 +74,9 @@ If you're happy with what you see, leave a comment on the GitHub issue stating y
 
 Pull requests are always welcome :simple_smile:
 
-If you're working on fixing a particular issue, referring to it in the description of your PR is really helpful for establishing the context needed for review.
+PRs should reference an open github issue marked with the [help wanted](https://github.com/fastlane/fastlane/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) label. Referring to the issue in the description of your PR is required and is really helpful for establishing the context needed for review.
 
-If your PR is contributing new functionality, did you create an issue to discuss it with the community first? Great! :raised_hands: Reference that issue in your PR.
+If you're contibuting new functionality, please open a new issue explaining the functionality desired first so that we can discuss as a community. We'll add the [help wanted](https://github.com/fastlane/fastlane/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) label if we believe this to be a meaningful contribution that will benefit other fastlane users and you go ahead with the pull request. :raised_hands:
```

Comment:
What do you think about reversing the order of this paragraph and the previous one? This one seems to be step 1.